README.md

```diff
@@ -30,165 +32,471 @@ The two guiding principles are:
+
+**`src/app.js`**
+
+```javascript
+import { render, Import } from 'hops';
+import React from 'react';
+
+const MyLazyComponent = Import('./my-component');
+
+export default render(
+  <div>
+    <h1>Hello World</h1>
+    <MyLazyComponent />
+  </div>
+);
+```
 
-By default Hops will find installed presets automatically (only when it is listed as a top-level dependency in your `package.json` - we won't attempt to find presets in dependencies of dependencies).
+### CLI / npm scripts
 
-If you prefer, you can also explicitly list the presets that you want to use under the `presets` key in your [application configuration](#configuration). This will disable automatic discovery of hops presets.
+The `hops` package acts as a CLI to be used inside [npm scripts]() or [npx]():
+
+**`package.json`**
 
 ```json
 {
-  "hops": {
-    "presets": ["hops-preset-defaults", "hops-react", "hops-redux"]
+  "scripts": {
+    "start": "hops start",
+    "develop": "hops develop",
+    "build": "hops build",
+    "serve": "hops serve"
   }
 }
 ```
 
-Some presets require (or allow) additional configuration. Read the sections below for each of the presets you are using to find out what options are available to you.
+#### `hops --help`
 
-### Available presets
+The Hops CLI provides short help texts for all its commands. If, for example, you want to see what options are available for the `build` command you can type: `npx hops build --help`.
 
-In addition to the core presets that [we mentioned earlier](#presets) these following presets are available to help you with the struggles of universal rendering, deployments, service workers and such.
+#### `npm start`
 
-#### `hops-react`
+This command will either start the development server or execute a production build and start the production server, depending on the environment variable `NODE_ENV` and/or the `--production` flag.
 
-This preset enables usage of React.js with JSX, React-Helmet and React-Router support in Hops applications.
+_The production flag (`--production` or `-p`) is a shortcut for setting the environment variable `NODE_ENV=production`, so these two are interchangable._
 
-Install it to your project:
+- `npm start` starts the development server with hot module reloading, etc.
+- `npm start --production` executes a production build and then starts the production server.
+- `npm start --production --static` exports static HTML pages for all configured [locations](#default-settings) and then starts the production server (_this only works in tandem with production mode_).
+
+#### `npm run develop`
```

Comment:
do we really have a `develop `-command in the `scripts`-block somewhere?

Reply:
removed it again, as `npm start` is sufficient.